client/app/routes/discussions.tsx

```diff
@@ -38,12 +38,23 @@ export const loader = async ({ request }: LoaderFunctionArgs) => {
 		orderBy: { createdAt: "desc" },
 	});
 
+	const userId = await checkAuth(request);
+	let notifications = [];
+	if (userId) {
+		notifications = await prisma.notificationSuscribers.findMany({
+			where: { userId, read: false },
+
+			include: { notification: true },
+		});
+	}
+	console.log(notifications?.length);
+
 	for (const post of posts) {
 		post.content = await renderSummary(post.content);
 	}
 
 	return json(
-		{ school: values.meta(), posts },
+		{ school: values.meta(), posts, notifications: notifications.length },
```

Comment:
Actually, the notifications count should be at the top level. In `root.tsx` preferably as the notifications widget will appear on all pages.

Reply:
Okay. I will work on it.